tests/lib/nested.sh

```diff
@@ -28,9 +28,15 @@ wait_for_service() {
 }
 
 wait_for_ssh(){
-    retry=400
+    local service_name="$1"
+    retry=800
     wait=1
     while ! execute_remote true; do
+        if ! systemctl show -p ActiveState "$service_name" | grep -q "ActiveState=active"; then
```

Comment:
will this race with the previous command `systemctl start`? I.e. maybe the ActiveState will be "activating" or something?

Reply:
`systemctl start` is synchronous by default and waits for the unit to start. The asynchronous version that doesn't wait requires `--no-block` flag which is not use there. So this shouldn't be racy.  Unless of course the svc-script is racy and is not executing/started when it first spawns things. So this looks ok to me.